### PR TITLE
Update all examples to ask users to make API tokens and use a default if not set by the user.

### PR DESCRIPTION
**Pull request by JasoonS**
In relation to adding this to the docs: https://github.com/enviodev/docs/pull/419

### CODE REVIEW

File: 1-python-fetch-all-logs/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 **Prerequisites**
 
-- Install `hypersync` and `asyncio` libraries using uv or pip: `uv pip install -r requirements`
+- Install `hypersync` and `asyncio` libraries using uv or pip: `uv pip install -r requirements.txt`
```
Comment (JasoonS) [inline 1-python-fetch-all-logs/README.md:5]: Just a typo

File: 10-ts-wallet-erc20-data/package-lock.json
Comment (JasoonS) [inline 10-ts-wallet-erc20-data/package-lock.json:1]: Ah, will amend commit and remove this file :+1:

File: 1-python-fetch-all-logs/fetch-logs.py [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
 import hypersync
 import asyncio
 
 # This simple script returns all logs from a contract within a block range
 async def main():
     # 35+ chains supports by hypersync see: https://docs.envio.dev/docs/overview-hypersync
-    client = hypersync.HypersyncClient(hypersync.ClientConfig(url="https://eth.hypersync.xyz"))
+    bearer_token = os.environ.get("HYPERSYNC_BEARER_TOKEN")
+    if not bearer_token:
+        print("\033[93mPlease create a token at https://envio.dev/app/api-tokens and set the HYPERSYNC_BEARER_TOKEN environment variable. API tokens will improve the reliability of the service, and in future they may become compulsory.\033[0m")
```
Comment (JasoonS) [inline 1-python-fetch-all-logs/fetch-logs.py:10]: Writes a message in noticeable yellow that the user should create an API token.

**Review by moose-code — APPROVED** [review]
This is fantastic thanks @JasoonS !

**Comment by moose-code** [comment]
Quite a chore but we maybe want to do the same for all client examples

**Pull request merged**